src/test/unit/base/services/authenticationService.test.ts

```diff
@@ -117,4 +120,54 @@ suite('AuthenticationService', () => {
     const ipFamily = await getIpFamily('https://dev.snyk.io');
     strictEqual(ipFamily, 6);
   });
+
+  test("Doesn't call setToken when token is empty", async () => {
+    const logAuthenticateButtonIsClickedFake = sinon.fake();
+    const analytics = {
+      logAuthenticateButtonIsClicked: logAuthenticateButtonIsClickedFake,
```

Comment:
You don't need to declare fake as separate variable - you can set sinon.fake() here directly

Reply:
In fact you can ditch this declaration completely by passing an empty interface. You don't need this object as you never reference it in the code.

Reply:
good catch, it makes sense to me (I copied it from the analytic test step, but I don't need an object).
done with 034c550cae23a798714ccc18d4ce8a60f5303560